From c8bc83f5eac49f083ae53d965577118c1b5c1116 Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Sun, 31 Oct 2010 22:29:27 +0000
Subject: Revert "fsck: add new -l switch to fsck mount options"

The '-l' option is already taken by fsck.ext[234]:
  "-l bad_blocks_file   Add to badblocks list"

This reverts commit cf1a105550766d2251bc10240b6058a37babd600 but keeps
the string changes.
---
diff --git a/src/fsck.c b/src/fsck.c
index fdcf6b4..dfaf221 100644
--- a/src/fsck.c
+++ b/src/fsck.c
@@ -142,7 +142,7 @@ static void test_files(void) {
 }
 
 int main(int argc, char *argv[]) {
-        const char *cmdline[8];
+        const char *cmdline[7];
         int i = 0, r = EXIT_FAILURE, q;
         pid_t pid;
         siginfo_t status;
@@ -213,7 +213,6 @@ int main(int argc, char *argv[]) {
         cmdline[i++] = "/sbin/fsck";
         cmdline[i++] = "-a";
         cmdline[i++] = "-T";
-        cmdline[i++] = "-l";
 
         if (!root_directory)
                 cmdline[i++] = "-M";
--
cgit v0.8.3-6-g21f6
